deploy/image-references

```diff
@@ -6,3 +6,19 @@ spec:
     from:
       kind: DockerImage
       name: docker.io/openshift/origin-cluster-samples-operator:latest
+  - name: jenkins
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-jenkins:latest
+  - name: jenkins-agent-base
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-jenkins-agent-base:latest
```

Comment:
i don't think we actually need or want the jenkins-agent-base reference.   right now i'm thinking we will not have an imagestream for it.

Reply:
agent-base removed